maquetado/scss/form.scss

```diff
@@ -0,0 +1,59 @@
+.info-container {
+  background-color: $wild-sand;
+  justify-content: space-between;
+  left: 50%;
+  max-width: 300px;
+  padding: 25px 25px 15px;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+  &::before {
+    content: '';
+    background-color: $cerulean;
+    height: 8px;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+}
+
+.login-img {
+  height: 70px;
+  margin: 0 auto 20px;
+  width: 210px;
+}
+
+
+.submit-button,
+.redirect-button {
+  border-radius: 5px;
+  display: flex;
```

Comment:
Add `align-items: center` to avoid this: 

![image](https://user-images.githubusercontent.com/36488701/59783574-6dffc200-928e-11e9-93ff-597ccd190539.png)